.dockerignore

```diff
@@ -10,6 +10,8 @@
 !az.completion
 # Make an exception for the license file
 !LICENSE.txt
+# Make an exception for docker temp files when build images
```

Comment:
```suggestion
# Make an exception for Azure CLI RPMs
```

Reply:
I apply this change to the PR.
`.dockerignore` is introduced in https://github.com/Azure/azure-cli/pull/4535, which aims to reduce the image size.
The whole file can be removed after we use `--mount`.